/*
 * Copyright (c) 2026 Element Creations Ltd.
 *
 * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial.
 * Please see LICENSE files in the repository root for full details.
 */

package io.element.android.features.wallet.api

import com.bumble.appyx.core.modality.BuildContext
import com.bumble.appyx.core.node.Node
import io.element.android.libraries.architecture.FeatureEntryPoint

interface WalletEntryPoint : FeatureEntryPoint {
    fun createNode(parentNode: Node, buildContext: BuildContext): Node

    interface NodeProxy {
        // Add methods if needed to wait for results
    }
}
